pkg/virt-config/feature-gates.go

```diff
@@ -32,8 +32,6 @@ const (
 	SidecarGate           = "Sidecar"
 	GPUGate               = "GPU"
 	SnapshotGate          = "Snapshot"
-	DataVolumesGate       = "DataVolumes"
-	SRIOVGate             = "SRIOVGate"
```

Comment:
@vladikr I am surpised that SRIOV is not gated. I thought that it involves some extra privileges.

Reply:
@rmohr seems like you've approved it: https://github.com/kubevirt/kubevirt/pull/2546

Reply:
@rmohr, afaik, there are no additional privileges with sr-iov, but we will lock 1G of memory for VMIs with SR-IOV.

Reply:
> @rmohr seems like you've approved it: #2546

ups

> @rmohr, afaik, there are no additional privileges with sr-iov, but we will lock 1G of memory for VMIs with SR-IOV.

 perfect that was what I was unsure about.